server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/shared/_schmodal.scss

```diff
@@ -122,7 +122,6 @@ $overlay-border-radius: 3px;
 
 .overlay-content {
   background: #fff;
-  position:   relative;
```

Comment:
This may have unintended consequences - is this really what you mean? it may be better to keep `relative` in place to establish a sensible offset parent, while being explicit about `overflow: visible`.